killbill-integration-tests/core/test_account.rb

```diff
@@ -132,5 +132,54 @@ def test_account_update
 
     end
 
+    def test_emails_notifications
+
+      account = create_account(@user, @options)
+
+      # Verify if response is success
+      assert(account.email_notifications(@options).response.kind_of? Net::HTTPSuccess)
+
+    end
+
+    def test_account_blocking_state
+
+      account = create_account(@user, @options)
+
+      # Verify account methods
+      assert_respond_to(account, :blocking_states)
+      assert_respond_to(account, :set_blocking_state)
+
+      # Get blocking states
+      blocking_states = account.blocking_states('ACCOUNT', nil, 'NONE', @options)
+
+      # Verify if the returned list is empty
+      assert(blocking_states.empty?)
+
+      # Verify if response is success
+      assert(account.set_blocking_state('STATE1', 'ServiceStateService', false, false, false, nil, @user, nil, nil, @options).response .kind_of? Net::HTTPSuccess)
+
+      # Verify if the returned list has now one element
+      blocking_states = account.blocking_states('ACCOUNT', nil, 'NONE', @options)
+      assert_equal(false, blocking_states.empty?)
+
+      # Verify blocking state fields
+      blocking_states = blocking_states.first
+      assert_equal('STATE1', blocking_states.state_name)
+      assert_equal('ServiceStateService', blocking_states.service)
+      assert_equal(false, blocking_states.block_change)
```

Comment:
Use `assert_false` instead (`assert_false(blocking_states.block_change)`).

Reply:
Done! Thanks for the tips!